server/db/models.py

```diff
@@ -298,3 +316,23 @@
     Column('game_id',       Integer, ForeignKey('game_stats.id')),
     Column('gametime',      Integer),
 )
+
+user_group = Table(
+    'user_group', metadata,
+    Column('id',                Integer,        primary_key=True),
+    Column('technical_name',    String(100),    nullable=False),
```

Comment:
Here `technical name` and `name_key` are shorter than before (`String(100)` vs `String(255)`).
That's probably long enough either way, but I wonder why they are different, given that they probably refer to keys in the same table of descriptions?

Reply:
Don't know, that's just how it is in the database :P

Reply:
Actually I should fix the `group_permission` ones. Those should be 100 as well I think.